app-admin/logrotate/logrotate-3.20.1-r1.ebuild

```diff
@@ -64,12 +64,14 @@ src_install() {
 	doman logrotate.8
 	dodoc ChangeLog.md
 
-	insinto /etc
+	insinto /usr/share/logrotate
```

Comment:
Can we add the comments for the Flatcar changes as to why they are needed?

Reply:
Added some comments, please let me know if they are enough.